File: C_Assignment/Assignment_33/programAQ3.c
///////////////////////////////////////////////////////////
//
//  File name : Assignement33Question3.c
//  Description : Write a program which accept one number from user and check whether
//                9th or 12th bit is on or off.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Input : 257
Output : TRUE

*/

#include<stdio.h>

typedef unsigned int UINT;
typedef int BOOL;

#define TRUE 1
#define FALSE 0

BOOL ChkBit(UINT iNo)
{
    UINT iMask = 0x00000900;

    if((iNo & iMask) != 0)
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    UINT iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter number: \n");
    scanf("%u",&iValue);

    bRet = ChkBit(iValue);

    if (bRet == TRUE)
    {
        printf("TRUE\n");
    }
    else
    {
        printf("FALSE\n");
    }
    
    return 0;
}

File: Java_Assignment/Assignment_60/programAQ3.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement60Question3.java
//  Description : Write a java program which print Fibonacci series
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

import java.util.Scanner;

public class programAQ3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter number of terms: ");
        int no = sobj.nextInt();

        int a = 0, b = 1;

        System.out.println("Fibonacci Series:");

        for (int i = 1; i <= no; i++)
        {
            System.out.print(a + " ");

            int next = a + b;
            a = b;
            b = next;
        }

        sobj.close();
    }
}